modules/appeals_api/lib/appeals_api/upload_error.rb

```diff
@@ -4,6 +4,7 @@
 
 module AppealsApi
   class UploadError < CentralMail::UploadError
+    # TODO: how does this work?
     STATSD_UPLOAD_FAIL_KEY = 'api.appeals.higher_level_review_upload.fail'
```

Comment:
Since this should be generic regardless of the document type uploaded, remove reference to HLR and use more generic naming like `document_upload.fail`.